### ISSUE TITLE
*gamma* and *color scale* not doing anything

### ISSUE THREAD

**Issue by kuadrat** [pudeIko/piva#28]
This has to do with the change in colormaps in `pyqtgraph` which was partially accounted for in a quick fix in 43c8558 and 9d76806.

Colormaps currently do not actually have any *gamma* or *vmax* functionality - it has to be properly reimplemented.

**Comment by pudeIko** [comment]
<img width="843" alt="image" src="https://user-images.githubusercontent.com/37271202/203581165-150b49eb-8721-4467-b3cc-47569d5c6b78.png">

Images not visible after commit. No errors in terminal.

**Comment by kuadrat** [comment]
Wow, how strange. For me it's working.
What's your version of pyqtgraph?

**Comment by kuadrat** [comment]
Can you try upgrading pyqtgraph to `0.13.1`?

**Comment by pudeIko** [comment]
It says it requires also upgrading python version. Which one are you using?

**Comment by pudeIko** [comment]
I upgraded and it works beautifully ;) 

I was keeping it at old version, because downgrading solved some problems in the past. In general I agree we should always aim to keep up with newest versions!

**Comment by kuadrat** [comment]
Cool, I have reflected this in e50f5bc1a962fde3910e56867eb0ee387dac3e24.

I used to fully agree with keeping things up to the latest versions, as this seems to be the de-facto standard for software development. However, the [section in the pyqtgraph documentation about using pyqtgraph in a larger project](https://pyqtgraph.readthedocs.io/en/latest/getting_started/how_to_use.html#embedding-pyqtgraph-as-a-sub-package-of-a-larger-project) makes a relevant point:

In a project where stability is more important than being up-to-date with the most recent features, it would be troublesome if the software stopped working just because something in one of the dependencies that is completely out of the control of the software developers changed. In this case, pyqtgraph actually suggests to "embed" the desired version into the larger project simply by copy-pasting it inside.

Maybe this is worth considering. This way, a sudden breaking of features like this colorscale issue could be prevented.
